packages/core/src/components/table/table.css

```diff
@@ -0,0 +1,109 @@
+.hds-table-container {
+  overflow-x: auto;
+}
+
+.hds-table {
+  border-collapse: collapse;
+  font-family: var(--font-default); /* Tartteeko laittaa? */
+  width: 100%;
+  line-height: 1.5;
+}
+
+.hds-table--dark {
+  --background-color: var(--color-bus);
```

Comment:
Can you elaborate more of the abstraction you had in mind?

Reply:
For example this could be:
```css
.hds-table--dark {
  --background-color: var(--background-color);
  }
 ```
 Then we could declare somewhere higher in the scope:
 ```css
  --background-color: var(--color-bus); 
 ```
In this way, the background color is already customizable without theming via CSS variables. It would also be in line with older components if we ever decide to continue with themes.

Reply:
Plaah, I think I have linked into the wrong line 🙈 
I think some of the borders could be part of the css variables 🤔

Reply:
Or maybe not, I'll leave this for you to consider 😁